jenkinsfiles/Jenkinsfile.grobid

```diff
@@ -5,7 +5,12 @@ elifePipeline {
         def tag = params.revision ? params.revision : 'latest'
 
         stage "Checkout", {
-            sh "rm -rf grobid && git clone -b ${revision} git@github.com:elifesciences/grobid.git"
+            sh(
+                "rm -rf grobid"
```

Comment:
multiple lines don't work here according to the [console](https://alfred.elifesciences.org/script)

Reply:
I don't have access to the console unfortunately. Is there an alternative multiple line syntax? Otherwise I can put it back on a single line, just seems very long.

Reply:
[Triple quotes](https://stackoverflow.com/questions/5079797/whats-wrong-with-groovy-multi-line-string#5079914)?

Reply:
I came across that. And `stripIndent` or `stripMargin` would be my next choice. But I don't understand why the version above shouldn't work (it's also one of the options). Would you mind giving me access to the console?

Reply:
Made you a Jenkins admin because running arbitrary code in the console allows to do anything in Jenkins, that's why it is restricted. Try the concatenation there, I was having some problems.

Reply:
I pasted this in the console and it seems to work (the brackets are important):

```groovy
def revision = "test"
println(
    "rm -rf grobid"
    + " && git clone -b catchall git@github.com:elifesciences/grobid.git"
    + " && cd grobid"
    + " && git checkout ${revision}"
)
```

Result:
```bash
rm -rf grobid && git clone -b catchall git@github.com:elifesciences/grobid.git && cd grobid && git checkout test
```

i.e. it does seem to work.

I don't know whether you have a preference relating to breaking the line up. This string concatenation seemed the more simple solution (no groovy knowledge required, apart from the brackets). `stripIndent` and `stripMargin` appear more magical. Although I might prefer that if multi-line strings are a common thing in the Jenkins config.

Definitely worth restricting the permissions. Happy to be demoted from the Jenkins admin role.

Reply:
I must have done something wrong when testing then

Reply:
Great, this now also worked with a commit: https://alfred.elifesciences.org/job/process/job/process-grobid-docker/30/console